lnwallet/btcwallet/btcwallet.go

```diff
@@ -554,10 +555,26 @@ func unminedTransactionsToDetail(
 		return nil, err
 	}
 
+	// Iterate over all the transaction outputs and  extract the Public Key of the
+	// destination address.
+	var destAddresses []btcutil.Address
+	for _, txOut := range wireTx.TxOut {
+		_, outAddresses, _, err :=
+			txscript.ExtractPkScriptAddrs(
```

Comment:
Nit: this line fits on the same one above.

Reply:
fixed